src/components/PoiItem.jsx

```diff
@@ -8,12 +8,15 @@ import Address from 'src/components/ui/Address';
 import classnames from 'classnames';
 import poiSubClass from 'src/mapbox/poi_subclass';
 import { capitalizeFirst } from 'src/libs/string';
+import { findBlock } from 'src/libs/pois';
+import StarsSummary from 'src/panel/poi/blocks/StarsSummary';
```

Comment:
Shouldn't `StarsSummary` be defined in `src/components`, like `OpeningHour` is for instance?